core/server/common/src/main/java/alluxio/master/MasterContext.java

```diff
@@ -21,14 +22,18 @@
 public final class MasterContext {
   private final JournalSystem mJournalSystem;
   private final SafeModeManager mSafeModeManager;
+  private final MetricsStore mMetricsStore;
 
   /**
    * @param journalSystem the journal system to use for tracking master operations
    * @param safeModeManager the manager for master safe mode
+   * @param metricsStore the metrics store for tracking cluster-level metrics
    */
-  public MasterContext(JournalSystem journalSystem, SafeModeManager safeModeManager) {
+  public MasterContext(JournalSystem journalSystem, SafeModeManager safeModeManager,
+      MetricsStore metricsStore) {
     mJournalSystem = Preconditions.checkNotNull(journalSystem, "journalSystem");
     mSafeModeManager = Preconditions.checkNotNull(safeModeManager, "safeModeManager");
+    mMetricsStore = Preconditions.checkNotNull(metricsStore);
```

Comment:
use the variable name as the second argument to Preconditions.checkNotNull

Reply:
done